package.json

```diff
@@ -77,7 +78,7 @@
   },
   "devDependencies": {
     "@architect/architect": "^10.16.3",
-    "@architect/plugin-lambda-invoker": "^1.2.0",
+    "@architect/plugin-lambda-invoker": "github:dakota002/plugin-lambda-invoker#8149c366f33af1e82ca2aeeda828c92ffc86e35c",
```

Comment:
Is there a PR for your changes to plugin-lambda-invoker?

Reply:
https://github.com/architect/plugin-lambda-invoker/pull/4 Yup, it may need some work, but I recently rebased it